stylesheet.css

```diff
@@ -0,0 +1,104 @@
+header{
+    display: flex;
+    padding: 10px 10px 10px 10px;
+    justify-content: space-around;
+    align-items: center;
+    border: 2px solid lightgray;
+    background-color: white;
+    border-radius: 5px;
+    margin-bottom: 10px;
+    margin-left: 0px;
+    margin-right: 0px; 
+}
+ul li{
+    text-decoration: none;
+    display: inline-block;
+    padding: 20px;
+
+}
+.button1{
+    cursor: pointer;
+    color: blue;
+    border: none;
+    font-size: large;
+    
+}
+
+
+.main-container{
+    width: 100%;
+    display: flex;
+    gap: 10px;
+    
+
+}
+.first-container{
+    width: 70%;
+}
+.item1{
+    width:100%;
+    height: 200px;
+    border: 2px solid lightgray;
+    background-color: white;
+    border-radius: 5px;
+
+}
+.item1 p, h2{
```

Comment:
same here, instead of adding suffix 1, 2 add something that indicates what the item is.
Same goes for everywhere.

Reply:
Resolved!